### PR TITLE
CLI: implement meta1 and meta2 rebuild operations

### PR DESCRIPTION
**Pull request by murlock**
##### SUMMARY
Wrap `oio-meta1-rebuilder` and `oio-meta2-rebuilder` within `openio-admin`

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
CLI Admin

##### SDS VERSION
```
openio-admin 5.0.0.0a1.dev48
```

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -152,7 +152,9 @@ openio.admin =
     election_status = oio.cli.election.election:ElectionStatus
     election_sync = oio.cli.election.election:ElectionSync
     meta1_decache = oio.cli.admin.service_decache:Meta1Decache
+    meta1_rebuild = oio.cli.admin.service_rebuild:Meta1Rebuild
     meta2_decache = oio.cli.admin.service_decache:Meta2Decache
+    meta2_rebuild = oio.cli.admin.service_rebuild:Meta2Rebuild
```
Comment (AymericDu) [inline setup.cfg]: We should use `repair` instead of `rebuild`.

Comment (AymericDu) [reply setup.cfg]: I confused with the items. `rebuild` is a good term for services.

File: oio/cli/admin/service_rebuild.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            '--workers',
+            type=int,
+            default=1,
+            help="Number of workers."
+        )
+        parser.add_argument(
+            '--items-per-second',
+            type=int,
+            default=30,
+            help="Max items per second per workers (30)."
```
Comment (fvennetier) [inline oio/cli/admin/service_rebuild.py]: per worker (without 's')

File: oio/cli/admin/service_rebuild.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        )
+        ifile_help = "Read container IDs from this file instead of redis. " \
+                     "Each line should be formatted like 'container_id'."
```
Comment (fvennetier) [inline oio/cli/admin/service_rebuild.py]: Each line should contain one container ID.

**Comment by AymericDu** [comment]
Replaced with PR #1740.

**Pull request closed without merging**